src/backend/Domain/Entities/Achievements.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Domain.Entities;
+
+public class Achievements : Entity
+{
+    public string Name { get; set; } = string.Empty;
+    public string Descriptions { get; set; } = string.Empty;
+}
```

Comment:
Adicionar propriedade navegadora para players. lembrando que deve ser uma relação many-to-many

Reply:
Foi adicionado a propriedade navegadora